lib/Backend/GlobOpt.cpp

```diff
@@ -2572,7 +2572,6 @@ GlobOpt::OptInstr(IR::Instr *&instr, bool* isInstrRemoved)
             if (this->RemoveFlowEdgeToFinallyOnExceptionBlock(instr))
             {
                 *isInstrRemoved = true;
-                this->currentBlock->RemoveInstr(instr);
```

Comment:
Can we move this whole block of EH handling code to a ProcessExceptionHandling() functions?  I'd like to remove code from OptInstr() to attempt to maintain readability.